web/source/resources/osk/kmwosk.css

```diff
@@ -272,8 +272,9 @@
 .kmw-title-bar-actions{position: absolute; cursor: default; right:2px; top:0; width: 33px; height: 13px;}
 .kmw-footer-caption{color:#fff;font:0.7em Arial;margin:0 0 0 4px;}
 
-.kmw-banner-bar{height:100%; width:100%; margin:0; background-color:darkorange; display: inline-block;}
-.kmw-banner-bar .kmw-suggest-option {background-color: orange; display:inline-block; text-align: center; height: 85%; border-radius: 5px;overflow-x:scroll}
+.kmw-banner-bar{height:100%; width:100%; margin:0; background-color:darkorange; display: inline-block; white-space: nowrap;}
```

Comment:
When we update the CSS, can we reformat it to multi-line and indented? There's no need for it to all be one-line like this (I would suggest a future update would be to use SASS so we can reduce the complexity immensely but that's not for today).